src/Zynga/Framework/Cache/V2/Driver/InMemory.hh

```diff
@@ -0,0 +1,126 @@
+<?hh // strict
+
+namespace Zynga\Framework\Cache\V2\Driver;
+
+use Zynga\Framework\StorableObject\V1\Interfaces\StorableObjectInterface;
+use Zynga\Framework\Dynamic\V1\DynamicClassCreation;
+use Zynga\Framework\Cache\V2\Driver\Base as DriverBase;
+use Zynga\Framework\Cache\V2\Interfaces\DriverConfigInterface;
+use Zynga\Framework\Cache\V2\Interfaces\DriverInterface;
+use Zynga\Framework\Cache\V2\Exceptions\StorableObjectRequiredException;
+use Zynga\Framework\Cache\V2\Driver\InMemory as InMemoryDriver;
+use Zynga\Framework\Exception\V1\Exception;
+
+/**
+ * Use this class when you want to mock memcache.
```

Comment:
So, there may be valid cases for only having an in memory cache vs memcache. I would say instead of saying `use this class when you want to mock Memcache` to have something like `This is an in-memory cache that will be cleared at the end of the process lifetime.`